pr_testing/test_multiple_prs.sh

```diff
@@ -938,24 +972,7 @@ if [ "X$TEST_CLANG_COMPILATION" = Xtrue -a $NEED_CLANG_TEST = true -a "X$CMSSW_P
 
   TEST_ERRORS=`grep -E "^gmake: .* Error [0-9]" $WORKSPACE/buildClang.log` || true
   GENERAL_ERRORS=`grep "^ALL_OK$" $WORKSPACE/buildClang.log` || true
-  get_compilation_warnings $WORKSPACE/buildClang.log > $WORKSPACE/all-warnings-clang.log
-  for i in $(get_warnings_files $WORKSPACE/all-warnings-clang.log) ; do
-    echo $i >> $WORKSPACE/clang-new-warnings.log
-    grep ": warning: " $WORKSPACE/all-warnings-clang.log | grep "/$i" >> $WORKSPACE/clang-new-warnings.log
-  done
-  if [ -e $WORKSPACE/clang-new-warnings.log ]  ; then
-    echo 'CLANG_NEW_WARNINGS;ERROR,Clang Warnings to fix,See Log,clang-new-warnings.log' >> ${RESULTS_DIR}/clang.txt
-    if $IS_DEV_BRANCH && [ $(echo "$IGNORE_BOT_TESTS" | tr ',' '\n' | grep '^CLANG-WARNINGS$' | wc -l) -eq 0 ] ; then
-      RUN_TESTS=false
-      ALL_OK=false
-      CLANG_BUILD_OK=false
-    fi
-  fi
 
-  if [ -d ${BUILD_LOG_DIR}/html ] ; then
-    mv ${BUILD_LOG_DIR}/html $WORKSPACE/clang-logs
-    echo 'CLANG_LOG;OK,Clang warnings summary,See Log,clang-logs' >> ${RESULTS_DIR}/clang.txt
-  fi
```

Comment:
@iarspider , I think we need to keep this block here otherwise `${BUILD_LOG_DIR}/html` will get delete and later re-generated for normal build